torch/csrc/jit/codegen/cuda/kernel_ir.cpp

```diff
@@ -173,18 +173,16 @@ std::vector<IterDomain*> TensorDomain::noBroadcasts(
   return no_broadcast_domains;
 }
 
-TensorView::TensorView(TensorDomain* domain, DataType dtype)
-    : Val(ValType::KirTensorView, dtype, true, true), domain_(domain) {}
-
 TensorView::TensorView(const fuser::TensorView* tv) : Val(tv), fuser_tv_(tv) {
-  domain_ = lowerValue(tv->domain())->as<TensorDomain>();
+  domain_ = lowerValue(tv->domain())->as<kir::TensorDomain>();
```

Comment:
please remove `kir::` (not needed, we're already in the kir namespace)

Reply:
There's a few `kir::` instances in this file, might want to look through them.